### PR TITLE
feat(cxl-lumo-styles): add social icons

### PR DESCRIPTION
**Pull request by anoblet**
https://app.clickup.com/t/86azkv8wk

### CODE REVIEW

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
@anoblet yes, we need X icon now. Twitter can stay as you already added it.

File: packages/cxl-lumo-styles/src/icons.js [hunk 1]
```diff
@@ -59,6 +59,15 @@ $documentContainer.innerHTML = `
...
+        </g>
+        <g id="youtube" transform="scale(.58824)" viewBox="0 0 71.412065 50">
+          <path d="M35.705078 0S13.35386.0001149 7.765625 1.4707031c-3 .8235294-5.4713925 3.2946921-6.2949219 6.3535157C.0001149 13.412454 0 25 0 25s.0001149 11.64637 1.4707031 17.175781c.8235294 3.058824 3.2360984 5.471393 6.2949219 6.294922C13.412684 50.000115 35.705078 50 35.705078 50s22.353171-.000115 27.941406-1.470703c3.058824-.82353 5.471393-3.236098 6.294922-6.294922 1.470588-5.588235 1.470703-17.175781 1.470703-17.175781s.058709-11.64614-1.470703-17.2343752c-.823529-3.0588236-3.236098-5.4713925-6.294922-6.2949219C58.058249-.00011488 35.705078 0 35.705078 0zm-7.117187 14.294922L47.175781 25l-18.58789 10.705078V14.294922z" transform="scale(1.7)"/>
+        </g>
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/src/icons.js:70]: Previously existing icons have width, height and viewBox with values around 20, 24, 60, here we have significantly bigger. Can we make them smaller? Also `transform="scale(.58824)"` looks weird.

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
X icon does not render.

Screenshot 2024-03-13 at 12 37 41

PS. @anoblet can you see icons on your localhost Storybook? I can't and never could. Only netlify version works.

File: packages/cxl-lumo-styles/src/icons.js (hunk 1 above)
Comment (anoblet) [reply packages/cxl-lumo-styles/src/icons.js]: I can't see the icons locally either :/ I was using the layout story and changing the icon in Chrome.

I fixed the X icon. It was just a matter of the fill color being set to white.

I tried to use the most authoritative source for the icons not in the Vaadin icon set:

Instagram - https://about.meta.com/brand/resources/instagram/instagram-brand/
X - https://about.x.com/en/who-we-are/brand-toolkit
YouTube - https://commons.wikimedia.org/wiki/File:YouTube_full-color_icon_(2017).svg#/media/File:YouTube_light_icon_(2017).svg

YouTube doesn't have an official SVG. I tried to find one that had a transparent triangle in the center.

I didn't touch anything from these SVG, and that's where the width, height, viewBox comes from. As long as the wrapping icon has a width, height set, they should render the correct size.

Comment (anoblet) [reply packages/cxl-lumo-styles/src/icons.js]: For YouTube, if someone has a subscription to Adobe Illustrator, we could generate our own SVG from official assets(https://www.youtube.com/howyoutubeworks/resources/brand-resources/#logos-icons-and-colors). They only distribute `ips`, `ai`, `png`. The triangle might need to be cut out.

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**